src/ReactNative/Components/ListView.purs

```diff
@@ -43,8 +49,11 @@ type ListViewPropsO section = ScrollViewPropsEx (
   , onChangeVisibleRows :: EventHandler2 RowMap RowMap
   , onEndReached :: EventHandler (Nullable ScrollEvent)
   , onEndReachedThreshold :: Int
-  , pageSize :: Int
+  -- , onChangeVisibleRows
+  , removeClippedSubviews :: Boolean
   , renderFooter :: Unit -> ReactElement
+  , stickySectionHeadersEnabled :: Boolean
+  , pageSize :: Int
```

Comment:
if this prop is available in required then it should not be in the optional props